packages/nns/src/types/governance_converters.ts

```diff
@@ -215,6 +215,10 @@ export interface OpenSnsTokenSwap {
     snsTokenE8s: bigint;
     maxParticipantIcpE8s: bigint;
     minIcpE8s: bigint;
+    neuron_basket_construction_parameters?: {
```

Comment:
I don't like that we have too, but shouldn't these be camel case to follow the existing pattern?

Reply:
yes, I'll change that.